meinberlin/assets/scss/components/_moderation_info.scss

```diff
@@ -0,0 +1,58 @@
+.moderation-info__accordion-title {
+    display: block;
+    position: relative;
+    padding: 0.8 * $padding 1.7 * $padding 0.8 * $padding $padding;
+    text-decoration: none;
+    color: inherit;
+
+    .moderation-info__accordion-chevron {
+        float: right;
+        transition: transform 0.3s;
+        line-height: inherit;
+    }
+
+    a[aria-expanded="true"] .moderation-info__accordion-chevron {
+        transform: rotate(180deg);
+    }
+}
+
+.moderation-info__accordion-body {
+    padding: 0 $padding $padding $padding;
+}
+
+.moderation-info__title {
+    display: inline-block;
+    margin: 0;
+}
+
+.moderation-info__btn {
+    float: right;
+    margin-right: $padding;
+}
+
+.moderation-info__section {
+    border-top: 1px solid $border-color;
+    padding-top: 1.4 * $padding;
+    margin-top: 0;
+}
+
+.moderation-info__list {
+    list-style: none;
+    padding-left: 0;
+    margin-bottom: 0;
+}
+
+.moderation-info__list-item {
+    border-top: 1px solid $border-color;
+    padding-top: 0.7 * $padding;
+    padding-bottom: 0.7 * $padding;
+    margin-bottom: 0;
+}
```

Comment:
padding is unequal at the bottom, you should be able to use `:las-child`selector to sort it